gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Fri Sep 18 13:17:45 ICT 2020
+#Tue Jan 12 10:52:22 ICT 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
-org.gradle.parallel=true
```

Comment:
Hmm. Why do we disable parallel on gradle build? 🤔

Reply:
hmm my AS automatically updates it when I the new gradle version. Maybe it is not needed anymore, let me check again 🙏